src/helper.ts

```diff
@@ -30,4 +36,88 @@ export function isSentioInternalError(err: any): boolean {
         return true;
     }
     return false;
-}
\ No newline at end of file
+}
+
+export async function getAllLPSnapshots(ctx : EthContext) {
+    // removes the suffix comprised of two letters coming from POINT_SOURCE
+    const snapshots = await ctx.store.list(AccountSnapshotLP)
+    const addresses = snapshots.map((snapshot) => snapshot.id.toString());
+    return {
+        snapshots,
+        addresses
+    }
+}
+export async function getAllYTSnapshots(ctx : EthContext) {
+    // removes the suffix comprised of two letters coming from POINT_SOURCE
+    const snapshots = await ctx.store.list(AccountSnapshotYT)
+    const addresses = snapshots.map((snapshot) => snapshot.id.toString());
+    return {
+        snapshots,
+        addresses
+    }
+}
+
+export async function getAllSYSnapshots(ctx : EthContext) {
+    // removes the suffix comprised of two letters coming from POINT_SOURCE
+    const snapshots = await ctx.store.list(AccountSnapshotSY)
+    const addresses = snapshots.map((snapshot) => snapshot.id.toString());
+    return {
+        snapshots,
+        addresses
+    }
+}
```

Comment:
looks like the comment is outdated, and a suggestion we could make these three functions into a generic one with arguments for ctx and schema to make the code more concise. 
```suggestion
export async function getAllLPSnapshots(ctx : EthContext) {
    const snapshots = await ctx.store.list(AccountSnapshotLP)
    const addresses = snapshots.map((snapshot) => snapshot.id.toString());
    return {
        snapshots,
        addresses
    }
}
export async function getAllYTSnapshots(ctx : EthContext) {
    const snapshots = await ctx.store.list(AccountSnapshotYT)
    const addresses = snapshots.map((snapshot) => snapshot.id.toString());
    return {
        snapshots,
        addresses
    }
}

export async function getAllSYSnapshots(ctx : EthContext) {
    const snapshots = await ctx.store.list(AccountSnapshotSY)
    const addresses = snapshots.map((snapshot) => snapshot.id.toString());
    return {
        snapshots,
        addresses
    }
}
```